### PR TITLE
Accommodate new RDKit

### PR DESCRIPTION
**Pull request by mwojcikowski**
Few things introduced by new RDKit, including one bugfix:

- Replace non-standard AA in test receptor to standard ones + adjust new test results
- Fix bug with checking if atom is in ring - current behavior is to include atoms in macromolecular ring in protein made by S-bridges, metal coordination etc. This might help with PLEC, need to check that; Current limit on ring size is up to 10 atoms in smallest ring to consider an atom to be a ring member in ECFP representation.
- Validate the zero-ordered metal coordination in new RDKit, currently monkey-patched them to single bonds - there is a problem with hydrogen atoms being added to residues coordinating these metal ions, which is probably not what we want to do. Need to validate how this impact our parsing stats from `RDKit-fixer`.

TODO:
- [x] Remove temporary tests, which helped debugging

Any other ideas?

### CODE REVIEW

File: oddt/fingerprints.py [hunk 1]
```diff
@@ -392,19 +393,27 @@ def _ECFP_atom_repr(mol, idx, use_pharm_features=False):
...
             if atom.GetAtomicNum() == 1:
                 raise Exception('ECFP should not hash Hydrogens')
             n_hs = atom.GetTotalNumHs(includeNeighbors=True)
+
+            # get ring info for atom and check rign size
+            isring = False
+            if atom.IsInRing():
+                # FIXME: this is not efficient, fixed by rdkit/rdkit#1859
+                isring = any(atom.IsInRingSize(size)
+                             for size in range(2, max_ring_size + 1))
```
Comment (marta-sd) [inline oddt/fingerprints.py]: 2 atoms in a ring? I don't think so :)

Comment (mwojcikowski) [reply oddt/fingerprints.py]: Indeed, it should be 3

**Comment by mwojcikowski** [comment]
Yes, this has changed, but I have no idea what is the current RDKit's maximum ring size. `max_ring_size = 10` was my arbitrary choice based on the expectation one have about rings in chemistry, rather than macro cycles.

I think we could revisit this issue after we merge this and #82

**Comment by mwojcikowski** [comment]
On the second thought - the max ring size wasn't changed on the RDKit's size, not in direct way. The rings are "broken" now by the zero-order bonds on metals. Anyhow, it's better to define the maximum ring size to consider for now. I've also fixed the iterative checking of "ring of size" membership on RDKit's side, see rdkit/rdkit#1859.

**Pull request merged**